src/LibVLCSharp.Avalonia.Unofficial/LibVLCSharp.Avalonia.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why create a new Unofficial package? IMO this csproj is not needed, we can use the current one https://github.com/videolan/libvlcsharp/blob/3.x/src/LibVLCSharp.Avalonia/LibVLCSharp.Avalonia.csproj